### DISCUSSION TITLE
Suggestions for improving the repo

### DISCUSSION THREAD

**Discussion by kaiwalyakoparkar** [Wittline/uber-expenses-tracking#4] [General]
Hey, thanks for sharing. This project is awesome. This are few suggestions from my side
1. Remove the unwanted categories which are not in use
<img src="https://i.imgur.com/BmuYoUz.png">
<img src="https://i.imgur.com/6leASKb.png">
2. Add important documents to make this project more inclusive
<img src="https://i.imgur.com/Dowv3LX.png">

**Comment by Wittline** [comment]
Hi @kaiwalyakoparkar, I have a question about your first suggestions: **Remove the unwanted categories which are not in use**, How I can remove these features from the repo?  "Releases and packages"

**Reply by kaiwalyakoparkar** [reply]
Go to <img src="https://i.imgur.com/Ou5c2x3.png">
and then disable which are not being used
 <img src="https://i.imgur.com/aSCqjhu.png">

**Reply by Wittline** [reply]
Ready, Thanks @kaiwalyakoparkar
